safe_transaction_service/history/indexers/erc20_events_indexer.py

```diff
@@ -102,18 +108,15 @@ def _process_decoded_element(self, event: EventData) -> EventData:
             of `value` or `tokenId`)
         """
         event_args = event["args"]
-        if (
-            "unknown" in event_args
-        ):  # Not standard event, trying to tell apart ERC20 from ERC721
-            logger.info(
-                "Cannot tell apart erc20 or 721 for token-address=%s - Checking token decimals",
-                event["address"],
-            )
-            value = event_args.pop("unknown")
-            if self._is_erc20(event["address"]):
-                event_args["value"] = value
-            else:
-                event_args["tokenId"] = value
+        if "unknown" in event_args:  # Not standard event
+            event_args["value"] = event_args.pop("unknown")
+
+        if self._is_erc20(event["address"]):
```

Comment:
Judging by line ~110 here in this file `event_args` is equal `event["address"]` , right? You could probably inline it here.

Reply:
No, `event_args = event["args"]`. I could inline in all the method, but I prefer to have a variable defined so I don't have a typo on `["args"]`